### ISSUE TITLE
Custom properties or .ddl for JdbcRepository

### ISSUE THREAD

**Issue by luca-bassoricci** [quarkiverse/quarkus-jberet#94]
Is there any way to specify custom properties or use a custom DDL for table creation?
I'm asking that because I want to create table with a specific schema, but I haven't found any solution yet.

**Comment by radcortez** [comment]
Yes, not at the moment in the way that the Quarkus integration is done (as you stated).

Are you up to push a PR for it? Thanks :)
